bin/nib-ls

```diff
@@ -22,6 +22,7 @@ import sys
 from math import ceil
 from optparse import OptionParser, Option
 from io import StringIO
+from collections import Counter
```

Comment:
Not compatible with Python 2.6.  We can drop Python 2.6 unless there's an obvious workaround.